### ISSUE TITLE
Fresh install doesnt work on browser

### ISSUE THREAD

**Issue by zrnecx** [expo/expo#5109]
> So ... forgive me if I'm being thicker than the offspring of a village idiot and a TV weathergirl....

Maybe im at wrong place here but...

I just installed expo-cli and created project with "expo init" and simply ran it with:
`expo start --web`

I want to develop in browser, but when i try to load it, i see this:
https://imgur.com/sXocdE8

I dont know what to do with it.

To say:
 My android and iphone devices works flawlessly, automatically refreshing etc, so its just problem with browser? what to do to fix it? 

sorry for stupid question, i know the fix will be obvious to many of you :D 🙈  🙉 🙊

**Comment by danielantelo** [comment]
same issue here, so i updated the babel config to:

```
module.exports = function(api) {
  api.cache(true);
  return {
    presets: [
      'babel-preset-expo',
      {
          web: { transformImportExport: true }
      }
    ],
  };
};
```

this then resulted in a different error

``` 
ReferenceError: [BABEL] /Users/d.antelo/.nvm/versions/node/v10.16.0/lib/node_modules/expo-cli/node_modules/react-dev-utils/webpackHotDevClient.js: Unknown option: .web. Check out https://babeljs.io/docs/en/babel-core/#options for more information about options.
```

Any ideas how to solve? I know expo web is not production ready, but eager to play around with it.

**Comment by brentvatne** [comment]
is this a 'blank' project? what exact steps did oyu follow to get here?

**Comment by danielantelo** [comment]
yes just a managed blank project, also tried the managed tabs version and same issue.

**Comment by zrnecx** [comment]
seems that  you added that "web"  part to config that is not valid... that is probably not the solution  to this problem.

**Comment by danielantelo** [comment]
i added that web part as per the docs on https://www.npmjs.com/package/babel-preset-expo as an attempt to solve the assigning exports error

**Comment by brentvatne** [comment]
@EvanBacon is looking into this

**Comment by EvanBacon** [comment]
**TL;DR:** Go to the bottom

When you run `npm --global list babel-preset-expo` if the version is less than 6.0.0 (expected) then it won't build correctly as the latest version contains a bunch of bug fixes. Really sorry for the inconvenience.

## The Problem

`babel-preset-expo@^5.2.0` was using some plugins that broke. [I fixed this in `babel-preset-expo@6.0.0`](https://github.com/expo/expo/pull/5012) but:
```
/Users/evanbacon/.nvm/versions/node/v10.14.1/lib
└─┬ expo-cli@3.0.6
  └─┬ @expo/xdl@56.1.3
    └─┬ @expo/webpack-config@0.7.0
      └── babel-preset-expo@5.2.0 😫
```

Luckily I already removed the dependency on `babel-preset-expo` here: https://github.com/expo/expo-cli/pull/861/files#r309024111 so this should be fixed in the next update (as long as your project is using `babel-preset-expo@6.0.0`). Again really sorry :( Currently working on a way to test `babel-preset-expo` (in expo/expo) with `@expo/webpack-config` (in expo/expo-cli) to prevent further mismatches.

## Quick Fix

Until I publish the latest you can do the following: 
1. `expo customize:web` 
2. Select `webpack.config.js`. This will force expo-cli to use a version of `@expo/webpack-config` that uses the local `babel-preset-expo@6.0.0`
3. Run as usual.

**Comment by zrnecx** [comment]
```
C:\Program Files\nodejs\global
`-- expo-cli@3.0.6
  `-- @expo/xdl@56.1.3
    `-- @expo/webpack-config@0.7.0
      `-- babel-preset-expo@5.2.0
```
Yes, that seems to be the problem here..., but:

```
C:\React\Zrny\webadmin>expo customize:web
ENOENT: no such file or directory, scandir ''
Set EXPO_DEBUG=true in your env to view the stack trace.
```
So i ran cmd

`set EXPO_DEBUG=true`

and only change is this

```
C:\React\Zrny\webadmin>expo customize:web
ENOENT: no such file or directory, scandir ''
Error: ENOENT: no such file or directory, scandir ''
```

**Comment by zrnecx** [comment]
So my solution was different, i just changed package.json:

added `"babel-preset-expo": "^6.0.0"` to dependencies and from there it works...

**Comment by danielantelo** [comment]
customize:web worked nicely. Up and running now, thanks.

**Comment by danielantelo** [comment]
hey @EvanBacon on the `expo init` -> managed -> tabs, freshly installed and with the custom `webpack.config` action that solved the blank project, getting the following error

```
ReferenceError: exports is not defined
Module.../../../@react-navigation/web/dist/createBrowserApp.js
```

Any ideas?

**Comment by richardpeng** [comment]
@danielantelo I also got that error using the `tabs` template for React Navigation. I was able to get past that by adding `sourceType: 'unambiguous'` to `babel.config.js`

```
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    sourceType: 'unambiguous'
  };
};
```

**Comment by EvanBacon** [comment]
Fixed in `expo-cli@3.0.8` (Remember to update with `npm i -g expo-cli`). Thank you all for your patience. If you customized, to work around this then you should be able to delete the `webpack.config.js` and remove `yarn remove -D @expo/webpack-config`.
